csrankings.csv

```diff
@@ -17640,7 +17656,7 @@ Søren Hauberg,DTU,https://www.compute.dtu.dk/english,NOSCHOLARPAGE
 Søren Ingvor Olsen,University of Copenhagen,http://diku.dk/english/staff/?pure=en/persons/5264,NOSCHOLARPAGE
 Søren Kimmer Schou Gregersen,DTU,https://www.compute.dtu.dk/english,NOSCHOLARPAGE
 Søren Lauesen,IT University of Copenhagen,https://pure.itu.dk/portal/en/persons/soeren-lauesen(ea6d10cf-6bb5-4414-8966-f1ed2591f794).html,NOSCHOLARPAGE
-Süleyman Cenk Sahinalp,Indiana University,https://www.soic.indiana.edu/all-people/profile.html?profile_id=291,NOSCHOLARPAGE
+Süleyman Cenk Sahinalp,University of Maryland - College Park,https://www.cs.umd.edu/people/sahinalp,O8wbTncAAAAJ

```

Comment:
Apparently not at Maryland either. https://ccr.cancer.gov/cancer-data-science-laboratory/s-cenk-sahinalp

Reply:
You are correct. I missed the recent affiliation of Prof. Sahinalp. I have corrected it to National Institute of Health in the csrankings.csv file.